sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -490,8 +493,11 @@ def locked_until_utc(self):
 
         :rtype: datetime.datetime
         """
-        if self._receiver._session_id or self.settled:  # pylint: disable=protected-access
-            return None
+        try:
+            if self.settled or self._receiver._session_id:  # pylint: disable=protected-access
```

Comment:
let's align the session check using attribute `self._receiver.session` instead of `self._receiver._session_id`?